lib/liquid/variable.rb

```diff
@@ -35,6 +35,52 @@ def raw
       @markup
     end
 
+    def format
+      output = Expression.format(name)
+
+      filters.each_index do |i|
+        output << " | "
+        output << filters[i][0]
+        if !filters[i][2].empty? || !filters[i][1].empty?
+          output << ':'
+        end
+        output << format_standard_filters(filters[i][1])
+        output << format_argument_filters(filters[i][2], filters[i][1].empty?)
+      end
+      "{{ #{output} }}"
+    end
+
+    def format_standard_filters(filters)
```

Comment:
IMO, `#format_standard_filters` and `#format_argument_filters` should be **`private`** methods.

Reply:
This is a good suggestion, there is likely a fair bit I can make private